components/atom/tag/src/Actionable/index.js

```diff
@@ -42,9 +42,11 @@ const ActionableTag = function ({
       {icon && iconPlacement === LEFT_ICON_PLACEMENT && (
         <span className="sui-AtomTag-icon">{icon}</span>
       )}
-      <span className="sui-AtomTag-label" title={title || label}>
-        {label}
-      </span>
+      {label ? (
```

Comment:
I'd rather update the condition to `label === ''` so that it displays something like a `0` in case `label` is `0`, for instance...